lib/services/lookup.js

```diff
@@ -482,6 +482,31 @@ function lookupServiceFactory(
     };
 
     /**
+     * Returns a list of object containing IP and Mac Addresses for the given Node ID.
+     * @param  {String} id Node ID
+     * @return {Promise.<Array>} A promise fulfilled with an array of
+     * objects containing IP Addresses and Mac Addresses associated with the Node.
+     */
+    LookupService.prototype.findIpMacAddresses = function (id) {
+        var self = this;
+        assert.isMongoId(id);
```

Comment:
Please remove this assertion, as RackHD will support multiple database, the nodeId will be changed to a common format  (such as uuid) rather than the ID that auto-generated by mongodb.